randovania/games/samus_returns/json_data/Area 4 - West.txt

```diff
@@ -308,7 +308,7 @@ Extra - asset_id: collision_camera_006
   > Door to Transport to Area 5 Entrance
       Trivial
   > Event - Chozo Seal Right Unlocked
-      After Area 4 (West) - Alpha Metroid+ and After Area 4 (East) - Gamma Metroid and After Area 4 (East) - Zeta Metroid
+      Metroid DNA
```

Comment:
Is this intended to be one dna, instead of 3?

Reply:
Yes it should be 3